pkg/k8shandler/kibana/reconciler.go

```diff
@@ -554,6 +554,9 @@ func newKibanaPodSpec(cluster *KibanaRequest, elasticsearchName string, proxyCon
 		fmt.Sprintf("-client-id=system:serviceaccount:%s:kibana", cluster.cluster.Namespace),
 		"-client-secret-file=/var/run/secrets/kubernetes.io/serviceaccount/token",
 		"-cookie-secret-file=/secret/session-secret",
+		"-cookie-refresh=10s",
```

Comment:
is there a matching test for this or is that a different proxy conf for maybe the es-proxy?  

otherwise /lgtm

Reply:
@jcantrill i think that is a different proxy conf